datadog/resource_datadog_logs_pipeline.go

```diff
@@ -0,0 +1,687 @@
+package datadog
```

Comment:
There are multiple logic within this file, conversion, constants, schemas, do we want to split into smaller/responsibility files?

Reply:
Check the existing files. It looks we are keeping a single file for each resource. cc @bkabrda